content/en/docs/concepts/extend-kubernetes/_index.md

```diff
@@ -163,6 +163,8 @@ After a request is authorized, if it is a write operation, it also goes through
 ) allow users to mount volume types without built-in support by having the
 Kubelet call a Binary Plugin to mount the volume.
 
+FlexVolume is deprecated in v1.23. Out-of-tree CSI driver is the recommended way to write volume drivers in Kubernetes. See this doc [here](https://github.com/kubernetes/community/blob/master/sig-storage/volume-plugin-faq.md#kubernetes-volume-plugin-faq-for-storage-vendors) for more information.
```

Comment:
```suggestion
FlexVolume is deprecated since Kubernetes v1.23. The Out-of-tree CSI driver is the recommended way to write volume drivers in Kubernetes. See [Kubernetes Volume Plugin FAQ for Storage Vendors](https://github.com/kubernetes/community/blob/master/sig-storage/volume-plugin-faq.md#kubernetes-volume-plugin-faq-for-storage-vendors) for more information.
```

Aside: Why is **Kubernetes Volume Plugin FAQ for Storage Vendors** not already part of the Kubernetes docs (maybe on https://kubernetes.dev/)?